game/graphics/mesh/submesh/staticmesh.cpp

```diff
@@ -4,7 +4,6 @@
 #include "gothic.h"
 
 StaticMesh::StaticMesh(const PackedMesh& mesh) {
-  static_assert(sizeof(Vertex)==sizeof(ZenLoad::WorldVertex),"invalid landscape vertex format");
   const Vertex* vert=reinterpret_cast<const Vertex*>(mesh.vertices.data());
```

Comment:
Removal static_assert is a good point. `reinterpret_cast` is also not needed, right?